test/core/comment.test.ts

```diff
@@ -0,0 +1,15 @@
+import { describe, it, expect } from 'bun:test';
+import { getAllComments } from '../../src/core/fetch/comment';
+import { chainId, daoAddress, apiUrl } from '../constants';
+
+describe('Comment', () => {
+  it('should get all comments', async () => {
+    const comments = await getAllComments({
+      chainId,
+      address: daoAddress,
+      slug: '2kexrO95CnUSOgKdGj_k1',
```

Comment:
how does `slug` or `proposalId` get generated? From frontend or database?

Reply:
`slug` is generated by the backend using `nanoid`